src/main/groovy/io/seqera/wave/controller/OpenAPIController.groovy

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Wave, containers provisioning service
+ *  Copyright (c) 2024, Seqera Labs
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.seqera.wave.controller
+
+import groovy.transform.CompileStatic
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.scheduling.TaskExecutors
+import io.micronaut.scheduling.annotation.ExecuteOn
+/**
+ * Implements a controller for openapi
+ *
+ * @author Munish Chouhan <munish.chouhan@seqera.io>
+ */
+@CompileStatic
+@Controller
+@ExecuteOn(TaskExecutors.BLOCKING)
+class OpenAPIController {
+    @Get(uri = "/openapi")
+    HttpResponse getOpenAPI() {
+    println("Redirecting to /openapi/")
+        HttpResponse.redirect(URI.create("/openapi/"))
+    }
```

Comment:
Let's move this here, that's done mostly for it 

https://github.com/seqeralabs/wave/blob/5d5fd91352744febfc010708e326467d24bef798/src/main/groovy/io/seqera/wave/controller/ServiceInfoController.groovy#L45-L60